### ISSUE TITLE
CSV/JSON files

### ISSUE THREAD

**Issue by dilemmax** [DIGITALCRIMINAL/OnlyFans#48]
Is it possible to remove any text in angle brackets and unicode text from the "text" column when scrapping to a json/csv file?

Currently, I open the json file in Word and use the wildcard search function to delete any text starting and ending with angle brackets, replace \n with a space etc. Then convert it to a spreadsheet and make the relevant changes before creating a batch file to rename the files with relevant titles.

Basically I have to do a lot before I end up with something like this

"ren "5d9ec0518452aaf05bd2d.mp4" "[2019-10-11] 194 - title.mp4""

**Comment by DIGITALCRIMINAL** [comment]
Do you mean you want file_name_format to be exported to the archive files?

**Comment by DIGITALCRIMINAL** [comment]
Yee, I'll do that in a bit. I'll message again when I've pushed the commit.

**Comment by dilemmax** [comment]
No worries. No rush on this though. It's just an enhancement. Thank you :)

**Comment by DIGITALCRIMINAL** [comment]
Will be implemented in the next release (Monday)
